src/jDatetime.php

```diff
@@ -437,8 +440,8 @@ private static function getDayNames($day, $shorten = false, $len = 1, $numeric =
 
     private static function getMonthNames($month, $shorten = false, $len = 3)
     {
-        $ret = '';
-        switch ( $month ) {
+        $ret = self::getMonth($month,'falaki');
```

Comment:
To make changes to this file, please send a pull request to the original project here: https://github.com/sallar/jDateTime
Direct changes should be reverted.